pkgs/applications/audio/easyabc/default.nix

```diff
@@ -1,7 +1,22 @@
-{ lib, fetchFromGitHub, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
+{ lib, fetchFromGitHub, fetchPypi, substituteAll, makeWrapper, python39, fluidsynth, soundfont-fluid, wrapGAppsHook, abcmidi, abcm2ps, ghostscript }:
 
-# requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
-python39.pkgs.buildPythonApplication {
+let
+  # requires python39 due to https://stackoverflow.com/a/71902541 https://github.com/jwdj/EasyABC/issues/52
+  python = python39.override {
+    packageOverrides = self: super: {
+      # currently broken with 4.2.1
```

Comment:
```suggestion
      self = python;
      # currently broken with 4.2.1
```

Reply:
Why?

Reply:
Can you explain why this is necessary or what this achieves?

Reply:
without this packageOverrides will use python where wxPython was not overwritten for other dependencies and will result in multiple wxPython versions being imported

Reply:
can't find example in <nixpkgs> to reproduce the issue, so maybe it's not necessary anymore, but was before

Reply:
ok found a case to reproduce, at this commit  800013956d54e37c4803a0e6aa2ed57bf29c107a

`nix-build -A awsebcli` without `self = localPython`

Reply:
Thanks, applied in ca3f0dfdc395dc4b207c51f0a7d316c37ebd0449

Reply:
suggestion was indeed in wrong line

Reply:
@FRidh can you comment if `self = python;` still necessary ? I tried a bunch of packages in nixpkgs, and they all evaluate to same derivation without `self`